workspaces/ballerina/ballerina-visualizer/src/views/GraphQLDiagram/index.tsx

```diff
@@ -341,22 +467,41 @@ export function GraphQLDiagram(props: GraphQLDiagramProps) {
                 />
             )}
             {isTypeEditorOpen && editingType && editingType.codedata.node !== "CLASS" && (
-                <PanelContainer
-                    title={`Edit Type${getTypeKindDisplayName(editingType?.codedata?.node) ?
-                        ` : ${getTypeKindDisplayName(editingType?.codedata?.node)}` :
-                        ''}`}
```

Comment:
We cannot remove this, as it is required to render the TypeEditor when a user needs to edit a type in graphql designer